user_interface/EbikeUIController/can.h

```diff
@@ -0,0 +1,68 @@
+typedef enum tagCAN_ID {
+    CAN_ID_BIKE_STATUS = 0x330,
+    CAN_ID_MOTOR_VOLTAGES = 0x331,
+    CAN_ID_MOTOR_REAL_CURRENTS = 0x332,
+    CAN_ID_MOTOR_CALC_VALUES = 0x333,
+    CAN_ID_UIC = 0x334,
+    CAN_ID_BMS_SOC = 0x355
+} CAN_ID;
+
+typedef struct tagCAN_BikeStatus_Struct{
+    uint16_t timestamp;
+    uint16_t speed;
+    uint8_t throttleInput; // receive throttleInput from motor controller for verification 
+    uint8_t errorWarning; // motor controller error/warning 
+    uint16_t temp_fet; // mosfet bridge temperature
+}CAN_BikeStatus_Struct;
```

Comment:
just a note, its good to have a different naming scheme for types and functions. When you change the function names, don't change the names of these types cause these are good 👍 .